src/baseball/main.py

```
@@ -1,10 +1,47 @@
-    """
-    # 프로그램의 메인 로직을 여기에 구현
+    while True:
+        arr = randnumber()
+
+        while True:
+            try:
+                num = int(input())  # 메시지 없이 입력받음
+                if num < 100 or num > 999:  # 3자리 숫자가 아닐 경우
+                    raise ValueError  # 🔴 명시적으로 예외 발생
+            except ValueError:
+                raise ValueError  # 🔴 숫자가 아니거나 3자리가 아닐 경우 예외 발생
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)